src/app/georeferencer/qgsgcplistwidget.cpp

```diff
@@ -136,6 +137,75 @@ void QgsGCPListWidget::itemClicked( QModelIndex index )
   mPrevColumn = index.column();
 }
 
+void QgsGCPListWidget::keyPressEvent( QKeyEvent *e )
+{
+  if ( e->key() == Qt::Key_Delete )
+  {
+    QModelIndex index = currentIndex();
+    if ( index.isValid() )
+    {
+      removeRow();
+      if ( model()->rowCount() > 0 )
+      {
+        setCurrentIndex( model()->index( index.row() == model()->rowCount() ? index.row() - 1 : index.row(), index.column() ) );
+      }
+    }
+    e->ignore();
```

Comment:
shouldn't this be accepted if the row was removed?

Reply:
You mean shouldn't this _not_ be ignored if a row is not removed?